images/linux/scripts/installers/aws.sh

```diff
@@ -7,12 +7,20 @@
 # Source the helpers
 source $HELPER_SCRIPTS/document.sh
```

Comment:
`source $HELPER_SCRIPTS/os.sh` should be added before start using `IsUbuntu20`

Reply:
Thank you @al-cheb! Fixed.